README.md

```diff
@@ -372,6 +372,23 @@ add-zsh-hook chpwd load-nvmrc
 load-nvmrc
 ```
 
+##### bash
+
+Put this into `$HOME/.bashrc` (or `$HOME/.bash_profile`, or whatever you use) to override the `cd` command to mimic the above zsh-hook:
+
+```bash
+# place this after nvm initialization!
+load-nvmrc() {
+  # same definition as zsh-hook
```

Comment:
i'd prefer to either repeat the definition, or (better), have it in one place, and then only have the different sections for zsh and bash?